### ISSUE TITLE
[bug]: occasionally happens escape character errors after flip pages

### ISSUE THREAD

**Issue by MyIpWasLocalhost** [MyIpWasLocalhost/flamepost#9]
ex:
before flip pages
<img width="491" alt="截屏2024-11-22 11 04 58" src="https://github.com/user-attachments/assets/9b56cb24-568e-4362-9163-049c2183c72c">
flip to next page and back:
<img width="497" alt="截屏2024-11-22 11 05 23" src="https://github.com/user-attachments/assets/3ae61058-b606-4a1c-9a3b-2e8f385a64ef">

**Comment by MyIpWasLocalhost** [comment]
caused by `function escapehtml(html)` being repeatedly called during flip pages in 
line 31-32, function `createPostDisplay()`, from _scripts_webpost.js_
```Javascript
    item.author = escapeHtml(item.author);
    item.content = escapeHtml(item.content);
```

**Comment by MyIpWasLocalhost** [comment]
fixed in [f302b01](https://github.com/MyIpWasLocalhost/flamepost/commit/f302b01f2f6da502ea9e88919f59548bc5e5abdd_)
